packages/mesos/README.md

```diff
@@ -1,3 +1,3 @@
 <h2>Patches to cherry-pick on top of open source Apache Mesos before building in DC/OS</h2>
 
-See [patches.json](patches.json) for the list of patches.
+See [extra/patches](patches) directory for the list of patches.
```

Comment:
I think we should simply kill this file. "extra/patches" is self-explanatory.

Reply:
I agree that `extra/patches` is self-explanatory but it's one level too deep and not visible to someone looking at this directory for Mesos sources.

Reply:
I think he meant
```
See [extra/patches](extra/patches) directory for the list of patches.
```
/me sneaks back into the shadows...

Reply:
I meant killing the file altogether. However now I think it makes sense to tell folks how to prepare new patches, e.g., use `git format-patch --stdout <range> | sed 1d` to omit the first line.

Reply:
I think something like the following should suffice:

<h2>Mesos Patches</h2>

We need to apply some patches on top of upstream Apache Mesos before building in
DC/OS.  The patches are stored in the [extra/patches] directory and applied as part of [build] script.

Here are the instructions to add/remove/update one or more patches:
* Checkout the upstream Mesos SHA.
* Apply all the cherry-picks that you need.
* Create patch files using `git format-patch -N <upstream-SHA>`
* Replace existing files in [extra/patches] with the new patche files.